src/msalexample-ios-macos/README.md

```diff
@@ -0,0 +1,102 @@
+products:
+- azure
+- azure-active-directory
+- ms-graph
+urlFragment: ms-identity-docs-code-app-swift-multiplatform
+---
+-->
+
+# Swift | Multiplatform SwiftUI iOS and macOS | user sign-in, protected web API access (Microsoft Graph) | Microsoft identity platform
+
+<!-- Build badges here
+![Build passing.](https://img.shields.io/badge/build-passing-brightgreen.svg) ![Code coverage.](https://img.shields.io/badge/coverage-100%25-brightgreen.svg) ![License.](https://img.shields.io/badge/license-MIT-green.svg)
+-->
+
+This Swift 5.5 Multiplatform SwiftUI iOS and macOS app authenticates a user and then makes a request to the Graph API as the authenticated user. The response to the request is presented to the user.
+
+![A screenshot of a Swift 5.5 Multiplatform SwiftUI iOS app displaying a response from Microsoft Graph.](./ios-signout-app.png)
+![A screenshot of a Swift 5.5 Multiplatform SwiftUI macOS app displaying a response from Microsoft Graph.](./macos-signout-app.png)
+
+## Prerequisites
+
+- Azure Active Directory (Azure AD) tenant and the permissions or role required for managing app registrations in the tenant.
+- Xcode 13.2.1
+- Swift 5.5
+
+## Setup
+
+### 1. Register the app
+
+First, complete the steps in [Register an application with the Microsoft identity platform](https://docs.microsoft.com/en-us/azure/active-directory/develop/tutorial-v2-ios#register-your-application) to register the application.
+
+Use these settings in your app registration.
+
+| App registration <br/> setting  | Value for this sample app                                           | Notes                                                                           |
+|--------------------------------:|:--------------------------------------------------------------------|:--------------------------------------------------------------------------------|
+| **Name**                        | `active-directory-swift-ios-macos-authorization-code-grant-flow`    | Suggested value for this sample. <br/> You can change the app name at any time. |
+| **Supported account types**     | **Accounts in this organizational directory only (Single tenant)**  | Suggested value for this sample.                                                |
+| **Platform type**               | **iOS / macOS **                                                    | Required value for this sample                                                  |
+| **Redirect URIs**               | `msauth.com.contoso.msalExample-macOS://auth` and `msauth.com.contoso.msalExample-iOS://auth` | Required value for this sample                    |
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+### 2. Open the project in Xcode 
+
+Next, open the _msalexamples-ios-macos.xcodeproj_ project in Xcode.
```

Comment:
:question: **Side question:** How was this project initialized? Is there an analog to `dotnet new $APP_TYPE` that you used for firing up a blank project that you then added the MSAL stuff to?

Asking for a friend who's never used Xcode. :flushed:

Reply:
just pure Xcode GUI.  

`Create a new Xcode project` -> `Multiplatform` -> `App`

or

`Create a new Xcode project` -> `Other` -> `Empty`